pkg/controller/secret/secret_controller.go

```diff
@@ -268,6 +271,10 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 
 		// https://issues.redhat.com/browse/OSD-10485
 		{Receiver: receiverMakeItWarning, Match: map[string]string{"alertname": "etcdHighNumberOfFailedGRPCRequests", "namespace": "openshift-etcd"}},
+
+		// https://issues.redhat.com/browse/OSD-11298
+		{Receiver: receiverMakeItCritical, MatchRE: map[string]string{"name": ".*master.*"}, Match: map[string]string{"alertname": "MachineWithoutValidNode", "namespace": "openshift-machine-api"}},
+		{Receiver: receiverMakeItCritical, MatchRE: map[string]string{"name": ".*master.*"}, Match: map[string]string{"alertname": "MachineWithNoRunningPhase", "namespace": "openshift-machine-api"}},
```

Comment:
I've also seen these alerts that contain a number of these alerts aggregated together ([for example](https://redhat.pagerduty.com/alerts/Q13AQ8A1QJUUK3)). I assume we dont need to worry about whatever is aggregating the alerts messing up our override route?

Reply:
the routing implimentaiton should be still be the same.